docs/import/manage_disaster.js

```diff
@@ -829,25 +853,37 @@ function createAssetDropdown(
     }
     select.append(assetOption);
   }
-  select.on(
-      'change', (event) => handleScoreAssetSelection(event, propertyPath));
 
   return select;
 }
 
 /**
- * Handles the user selecting an asset for one of the possible score types.
- * @param {Event} event Event, used to identify the target
+ * Adds the default change handler, which updates our internal data (and
+ * Firestore) when this element changes.
+ * @param {JQuery<HTMLElement>} elt
+ * @param {Array<string>} propertyPath The path to the value of this element
+ * @return {JQuery<HTMLElement>} The passed-in element, for chaining
+ */
+function addChangeHandler(elt, propertyPath) {
```

Comment:
nit: can we make this function name more specific like addScoreAssetChangeHandler

Reply:
the map bounds aren't "assets" per se, but are handled here. Changed name here and below to use "asset data", which is still kind of a lie, I guess.